portiaui/app/utils/browser-features.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+const { Promise } = Ember.RSVP;
+
+export default function hasBrowserFeatures() {
+    let features = Ember.A(['flexbox']);
```

Comment:
We have the ember prototype extensions enabled, and use native arrays throughout